v2/lib/webpack/server.js

```diff
@@ -5,7 +5,6 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-const _ = require('lodash');
```

Comment:
Technically we can now remove lodash from the `package.json` but let's keep it around as I think it will be useful in future.